### PR TITLE
ported minrep to api2

### PR DESCRIPTION
**Pull request by cseed**

### CODE REVIEW

File: python/hail/methods/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
     import_interval_list, import_bed, import_fam, grep, import_bgen, import_gen, import_table, \
     import_plink, read_matrix, read_table, get_vcf_metadata, import_vcf, index_bgen
 from .statgen import linreg, sample_rows, ibd, ld_matrix, grm, rrm, pca, hwe_normalized_pca, \
-    pc_relate, split_multi_hts, balding_nichols_model, FilterAlleles
+    pc_relate, split_multi_hts, balding_nichols_model, FilterAlleles, min_rep
```
Comment (jigold) [inline python/hail/methods/__init__.py]: This needs to get added to `index.rst` for methods.

File: python/hail/methods/statgen.py [hunk 2]
```diff
@@ -1299,3 +1299,36 @@ def filter(self):
...
+    left_aligned : bool
+        If ``True``, variants are assumed to be left aligned and have
+        unique loci.  This avoids a shuffle.  If the assumption is
+        violated, an error is generated.
+
+    Returns
+    -------
+    :class:`.MatrixTable`
+    """
+    return MatrixTable(ds._jvds.minRep(left_aligned))
```
Comment (jigold) [inline python/hail/methods/statgen.py]: Are there any tests for this that need to be ported over?

File: python/hail/methods/__init__.py (hunk 1 above)
Comment (cseed) [reply python/hail/methods/__init__.py]: Done.

File: python/hail/methods/statgen.py (hunk 2 above)
Comment (cseed) [reply python/hail/methods/statgen.py]: No :-/

Comment (jigold) [reply python/hail/methods/statgen.py]: Can you add just the function call to the tests with a FIXME to make it an actual test with asserts?

Comment (cseed) [reply python/hail/methods/statgen.py]: Done.

Comment (jigold) [reply python/hail/methods/statgen.py]: oh I see you added it already.

**Pull request merged**